velox/docs/develop/scalar-functions.rst

```diff
@@ -543,7 +543,10 @@ Vector Functions
 
 Simple functions process a single row and produce a single value as a result.
 Vector functions process a batch or rows and produce a vector of results.
-Some of the defining features of these functions are:
+When implementing a function, simple function is preferred unless the implementation
```

Comment:
Thanks. Perhaps, update https://github.com/facebookincubator/velox/blob/main/CONTRIBUTING.md#adding-scalar-functions as well.

Reply:
Updated. Thanks for the review.